ansible/templates/vbmcd.service.j2

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Virtual BMC daemon
+
+[Service]
+Type=simple
+Restart=on-failure
+ExecStart="{{ virtualenv_path }}/bin/vbmcd" --foreground
```

Comment:
vbmc_virtualenv_path?

Reply:
Is this service able to stop the daemon?

Reply:
I meant to change this as well...

I should probably start reviewing my pull requests using GitHub's diff viewer before adding reviewers to them, might stop me making PRs with so many stupid mistakes.

Reply:
But yes, `systemctl stop vbmcd` and `systemctl start vbmcd` seem to work fine.